ingest/src/main/java/com/connexta/ingest/exceptions/StorageException.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.ingest.exceptions;
+
+public class StorageException extends RuntimeException {
```

Comment:
❓Why does this extend RuntimeException?

Reply:
I was basing it off of the S3 exceptions that were thrown. Do you know a better option?

Reply:
🤷‍♀ That seems like a good enough reason to me.